guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Renewing_a_Custom_SSL_Certificate_{smart-proxy-context}_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+[NOTE]
+====
+If your {SmartProxy} is already configured with CA-signed SSL certificates and has been renewed by the same Certificate Authority, there is no need to replace the consumer package `katello-ca-consumer-latest.noarch.rpm` on clients registered to {SmartProxy}.
```

Comment:
"there is no need" > "you do not have to ..."

Reply:
maybe also "SmartProxyServer"?

Reply:
Agree :+1: